diff --git a/src/dsysinfo.cpp b/src/dsysinfo.cpp
index 44960ff..28c46ef 100644
--- a/src/dsysinfo.cpp
+++ b/src/dsysinfo.cpp
@@ -415,6 +415,12 @@ void DSysInfoPrivate::ensureReleaseInfo()
         productType = DSysInfo::UnknownType;
     } else {
         switch (productTypeString.at(0).unicode()) {
+	case 'g':
+	case 'G':
+            if (productTypeString.compare("gentoo", Qt::CaseInsensitive) == 0) {
+                productType = DSysInfo::Gentoo;
+            }
+            break;
         case 'd':
         case 'D':
             if (productTypeString.compare("deepin", Qt::CaseInsensitive) == 0) {
diff --git a/src/dsysinfo.h b/src/dsysinfo.h
index 136652e..c0e7e98 100644
--- a/src/dsysinfo.h
+++ b/src/dsysinfo.h
@@ -43,7 +43,8 @@ public:
         openSUSE,
         SailfishOS,
         Ubuntu,
-        Uos
+        Uos,
+        Gentoo
     };
 
     enum DeepinType {
