diff --git a/src/frame/CMakeLists.txt b/src/frame/CMakeLists.txt
index 24b6fec..d971e13 100644
--- a/src/frame/CMakeLists.txt
+++ b/src/frame/CMakeLists.txt
@@ -264,7 +264,6 @@ set(ACCOUNTS_FILES
                 modules/accounts/usermodel.cpp
                 window/modules/accounts/accountsmodule.cpp
                 window/modules/accounts/accountswidget.cpp
-                window/modules/accounts/pwqualitymanager.cpp
                 window/modules/accounts/createaccountpage.cpp
                 window/modules/accounts/modifypasswdpage.cpp
                 window/modules/accounts/accountsdetailwidget.cpp
@@ -653,8 +652,6 @@ if (NOT KF5_QT_LIBRARIES)
 endif()
 # end KF5 network manager
 
-set(DEEPIN_PW_CHECK libdeepin_pw_check.so)
-
 function(add_build_macro macro)
     string(TOUPPER ${macro} macro)
     add_definitions(-D${macro})
@@ -783,7 +780,6 @@ target_link_libraries(${BIN_NAME} PRIVATE
     ${LibNM_LIBRARIES}
     ${KF5_QT_LIBRARIES}
     ${Qt5Svg_LIBRARIES}
-    ${DEEPIN_PW_CHECK}
     crypt
     ${LIBS}
     PolkitQt5-1::Agent
diff --git a/src/frame/window/modules/accounts/createaccountpage.cpp b/src/frame/window/modules/accounts/createaccountpage.cpp
index 7a4fd10..85aef03 100644
--- a/src/frame/window/modules/accounts/createaccountpage.cpp
+++ b/src/frame/window/modules/accounts/createaccountpage.cpp
@@ -23,7 +23,6 @@
 #include "widgets/titlelabel.h"
 #include "window/utils.h"
 #include "groupitem.h"
-#include "pwqualitymanager.h"
 
 #include <DFontSizeManager>
 #include <DDesktopServices>
@@ -502,7 +501,7 @@ bool CreateAccountPage::checkPassword(DPasswordEdit *edit)
         }
     }
 
-    PwqualityManager::ERROR_TYPE error = PwqualityManager::instance()->verifyPassword(m_nameEdit->lineEdit()->text(),
+    /* PwqualityManager::ERROR_TYPE error = PwqualityManager::instance()->verifyPassword(m_nameEdit->lineEdit()->text(),
                                                                                       edit->lineEdit()->text());
 
     if (error != PwqualityManager::ERROR_TYPE::PW_NO_ERR) {
@@ -543,7 +542,7 @@ bool CreateAccountPage::checkPassword(DPasswordEdit *edit)
         }
 
         return false;
-    } else {
+    } else */{
         edit->setAlert(false);
         edit->hideAlertMessage();
     }
diff --git a/src/frame/window/modules/accounts/modifypasswdpage.cpp b/src/frame/window/modules/accounts/modifypasswdpage.cpp
index 1affaad..e5ba487 100644
--- a/src/frame/window/modules/accounts/modifypasswdpage.cpp
+++ b/src/frame/window/modules/accounts/modifypasswdpage.cpp
@@ -21,12 +21,9 @@
 
 #include "modifypasswdpage.h"
 #include "widgets/titlelabel.h"
-#include "pwqualitymanager.h"
 #include "../../utils.h"
 #include "createaccountpage.h"
 
-#include "deepin_pw_check.h"
-
 #include <DFontSizeManager>
 #include <DDBusSender>
 
@@ -156,8 +153,9 @@ void ModifyPasswdPage::clickSaveBtn()
 
 void ModifyPasswdPage::onPasswordChangeFinished(const int exitCode, const QString &errorTxt)
 {
-    PwqualityManager::ERROR_TYPE error = PwqualityManager::instance()->verifyPassword(m_curUser->name(),
+    /* PwqualityManager::ERROR_TYPE error = PwqualityManager::instance()->verifyPassword(m_curUser->name(),
                                                                                       m_newPasswordEdit->lineEdit()->text());
+    */
     if (exitCode != 0) {
         if (errorTxt.startsWith("Current password: passwd:")) {
             m_oldPasswordEdit->setAlert(true);
@@ -171,16 +169,17 @@ void ModifyPasswdPage::onPasswordChangeFinished(const int exitCode, const QStrin
             return;
         }
 
-        if (error == PW_NO_ERR) {
+        // if (error == PW_NO_ERR) {
             if (m_newPasswordEdit->lineEdit()->text() != m_repeatPasswordEdit->lineEdit()->text()) {
                 m_repeatPasswordEdit->setAlert(true);
                 m_repeatPasswordEdit->showAlertMessage(tr("Passwords do not match"), m_repeatPasswordEdit, 2000);
                 return;
             }
-        }
+        //}
 
-        m_newPasswordEdit->setAlert(true);
-        m_newPasswordEdit->showAlertMessage(PwqualityManager::instance()->getErrorTips(error));
+        //m_newPasswordEdit->setAlert(true);
+        //m_newPasswordEdit->showAlertMessage(PwqualityManager::instance()->getErrorTips(error));
+/*
         // 企业版控制中心修改密码屏蔽安全中心登录安全的接口需求
         if ((DSysInfo::uosEditionType() == DSysInfo::UosEnterprise) || (DSysInfo::uosEditionType() == DSysInfo::UosEnterpriseC))
             return;
@@ -213,6 +212,7 @@ void ModifyPasswdPage::onPasswordChangeFinished(const int exitCode, const QStrin
             });
             dlg.exec();
         }
+*/
     } else {
         Q_EMIT requestBack();
     }
